src/it/scala/org/ergoplatform/it/Docker.scala

```diff
@@ -152,7 +154,7 @@ class Docker(suiteConfig: Config = ConfigFactory.empty, tag: String = "ergo_inte
     val networkingConfig = ContainerConfig.NetworkingConfig
       .create(Map(networkName -> endpointConfigFor(ip)).asJava)
 
-    val knownPeersSetting = seedAddress.fold("")(sa => s" -Dscorex.network.knownPeers.0=$sa")
+    val knownPeersSetting = knownPeers(nodes, nodeConfig)
```

Comment:
we need `knownPeers ` function in all integration tests and it always almost the same. I think it worth adding default implementation for this method at least. There is no need for making it abstract here